pipelines/HSC/DRP-RC2.yaml

```diff
@@ -341,12 +296,10 @@ subsets:
   step6:
     subset:
       - consolidateDiaSourceTable
-      - writeRecalibratedSourceTable
-      - transformSourceTable
       - consolidateSourceTable
     description: >
       Tasks that can be run together, but only after the 'step1', 'step2',
-      'step3', and 'step4' subsets
+      'step3', and 'step4', 'step5' subsets
 
       This step includes visit-level aggregation tasks. Running without tract
       or patch in the data query is recommended, otherwise the outputs of
```

Comment:
```suggestion
      or patch in the dataquery is recommended, otherwise the outputs of
```

Reply:
Yeah, I was inconsisten. Looked up the pipetask run docs:
```  -d, --data-query QUERY          User data selection expression.
```
I'm going to change them all to `data query` (which is what we had written before)